docs/quantum/tutorials/basic_qaoa.rst

```diff
@@ -79,7 +79,7 @@ Now let's write a function that accepts our graph object and uses QAOA to try to
     :end-at: qaoa = QAOA.default(cost_hamiltonian=hamiltonian, n_layers=2)
 
 Notice we didn't have to build up the Hamiltonian or ansatz ourselves, Orquestra Core just did that for us! MaxCut isn't the only problem we can do that for, either. The ``opt`` part of Orquestra Core has `a number of problems <https://github.com/zapatacomputing/orquestra-opt/tree/main/src/orquestra/opt/problems>`_ readily solvable like `graph partition <https://github.com/zapatacomputing/orquestra-opt/blob/main/src/orquestra/opt/problems/graph_partition.py>`_ and `vertex cover <https://github.com/zapatacomputing/orquestra-opt/blob/main/src/orquestra/opt/problems/vertex_cover.py>`_. Similarly, we can select from a `number of ansatzes <https://github.com/zapatacomputing/orquestra-vqa/tree/main/src/orquestra/vqa/ansatz>`_.
-If you are familiar with QAOA, you might have noticed, that we did not specify a lot of details, such as what ansatz or optimizer do we want to use. That's because in this tutorial we will rely on the "default" version of QAOA which `QAOA`(TODO) class provides. For more details on how to use it, please refer to VQA guide (TODO).
+If you are familiar with QAOA, you might have noticed, that we did not specify a lot of details, such as what ansatz or optimizer do we want to use. That's because in this tutorial we will rely on the "default" version of QAOA which `QAOA <https://docs.orquestra.io/docs/api/orquestra/vqa/algorithms/qaoa/index.html#module-orquestra.vqa.algorithms.qaoa>`_ class provides. For more details on how to use it, please refer to `VQA guide <https://docs.orquestra.io/docs/core/quantum/guides/vqa_guide/introduction.html>`_.
```

Comment:
issue: rather than inserting a static link, it would be better to insert a link that will change based on the context like what's done in the [circuit_runners guide](https://raw.githubusercontent.com/zapatacomputing/orquestra-core/2e9112c44b389d7f1a03e8b707ef4e85c4097d84/docs/quantum/guides/circuit_runners.rst)

Reply:
I assume it is for QAOA class, but I keep a static link to the VQE tutorial, right?